### PR TITLE
Update UAC component

### PR DESCRIPTION
**Pull request by rmccar**
- Update component to latest styling and content
- Add error example

### CODE REVIEW

**Comment by rmccar** [comment]
[Update UAC component](https://trello.com/c/E8hMuNS2/520-update-uac-component)

File: src/components/uac/examples/uac-error/index.njk [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            "text": "Enter an access code to continue"
+        }
+    }) }}
+
+    {{ onsButton({
+        "text": "Access survey",
+        "classes": "u-mb-xl"
+    }) }}
+
+    <h2 class="u-fs-r--b u-mb-no">Don't have a code?</h2>
```
Comment (jrbarnes9) [inline src/components/uac/examples/uac-error/index.njk:60]: Can you update this block to match end to end proto but change content to for a generic 'survey' example?...

Screenshot 2020-06-16 at 15 03 02

File: src/components/uac/examples/uac-error/index.njk [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                    "index": true,
+                    "url": "#uac",
+                    "classes": "js-inpagelink"
+                }
+            ]
+        })
+    }}
+    {% endcall %}
+
+    {{ onsUACInput({
```
Comment (jrbarnes9) [inline src/components/uac/examples/uac-error/index.njk:39]: All text inputs within error panels should have a red border until focussed... 
Screenshot 2020-06-16 at 15 05 17

File: src/patterns/uac/index.njk [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Unique Access Codes
+title: Access Codes
 group: Ask users for...
 ---
 {% from "views/partials/example/_macro.njk" import patternlibExample %}
```
Comment (jrbarnes9) [inline src/patterns/uac/index.njk:5]: Add a 'This pattern requires documentation.' panel under patter page heading, until we can document this properly

File: src/components/uac/examples/uac-error/index.njk [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            "text": "Enter an access code to continue"
+        }
+    }) }}
+
+    {{ onsButton({
+        "text": "Access survey",
+        "classes": "u-mb-xl"
+    }) }}
+
+    <h2 class="u-fs-r--b u-mb-no">If you don't have an access code?</h2>
```
[the remark this answers is not in the document]
Comment (jrbarnes9) [reply src/components/uac/examples/uac-error/index.njk:61]: Update in both examples

**Comment by jrbarnes9** [comment]
[Update UAC component](https://trello.com/c/E8hMuNS2/520-update-uac-component)

File: src/components/uac/examples/uac-error/index.njk [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+{% block main %}
+    {% call
+        onsPanel({
+            title: "This page has an error",
+            type: "error"
+        })
+    %}
+
+    <h1 class="u-mt-l">Start survey</h1>
```
[the remark this answers is not in the document]
Comment (rmccar) [reply src/components/uac/examples/uac-error/index.njk:37]: The h1 is after the onsPanel, line 39 is the uac input

**Comment by jrbarnes9** [comment]
[Issue #782: Highlight empty mandatory text inputs with a red outline](https://trello.com/c/PYQ082rm/528-issue-782-highlight-empty-mandatory-text-inputs-with-a-red-outline)

File: src/components/uac/_macro.njk [hunk 6]
```diff
@@ -1,23 +1,39 @@
...
-            "data-group-size": groupSize,
-            "autocomplete": "off"
-        }
-    }) }}
+    {% set content %}
+        {{ onsInput({
+            "id": params.id,
+            "type": params.type,
+            "name": params.name,
+            "classes": "uac__input js-uac" + (" u-mb-s" if params.postTextboxLinkText else ""),
```
Comment (jrbarnes9) [inline src/components/uac/_macro.njk:14]:
```suggestion
            "classes": "uac__input js-uac" + (" u-mb-xs" if params.postTextboxLinkText else ""),
```
...spacing too large.

**Comment by rmccar** [comment]
[Issue #781: Padding not being applied to input boxes](https://trello.com/c/ANuacfGt/526-issue-781-padding-not-being-applied-to-input-boxes)

**Pull request merged**